src/datatable/tpl/layout.tpl

```diff
@@ -85,15 +85,7 @@
                                     {{#if id}}
                                         {{#with ../../../../this}}
                                             {{#unless ../hidden}}
-                                                {{#if ../../disabled}}
-                                                    {{#with ../../../this}}
-                                <button class="btn-info small {{id}}"{{#if title}} title="{{title}}"{{/if}} disabled="disabled">{{#if icon}}<span class="icon-{{icon}}"></span> {{/if}}{{#if label}} {{label}}{{/if}}</button>
-                                                    {{/with}}
-                                                {{else}}
-                                                    {{#with ../../../this}}
-                                <button class="btn-info small {{id}}"{{#if title}} title="{{title}}"{{/if}}>{{#if icon}}<span class="icon-{{icon}}"></span> {{/if}}{{#if label}} {{label}}{{/if}}</button>
-                                                    {{/with}}
-                                                {{/if}}
+                                <button class="btn-info small {{../../id}}"{{#if ../../title}} title="{{../../../title}}"{{/if}}{{#if ../../disabled}} disabled="disabled"{{/if}}>{{#if ../../icon}}<span class="icon-{{../../../icon}}"></span>{{/if}} {{../../label}}</button>
```

Comment:
Do we have a limit for line length? This seems too big to be on a single line and can be broken down in multiple to make it more readable :)

Reply:
Indeed, we could break in several lines. Usually in templates we tend to allow long lines, but anyway, I will apply some reformatting ;)